core/shared/platform/windows/shared_platform.cmake

```diff
@@ -13,9 +13,7 @@ include_directories(${PLATFORM_SHARED_DIR}/../include)
 file (GLOB_RECURSE source_all ${PLATFORM_SHARED_DIR}/*.c
 			      ${PLATFORM_SHARED_DIR}/*.cpp)
 
-if (NOT WAMR_BUILD_LIBC_WASI EQUAL 1)
-    list(REMOVE_ITEM source_all ${PLATFORM_SHARED_DIR}/win_file.c)
```

Comment:
Similar to the previous comment, I'm not completely sure why we're changing this? This file only has the implementation of functions used in wasi libc so compiling it isn't necessary when it isn't enabled and it means we have to add a stub for `convert_errno`.

Reply:
See above.

Reply:
Had better not change this file, convert_errno is already implemented in libc_errno.c?

Reply:
Yes, but that means we must compile it unconditionally .